app/layout.tsx

```diff
@@ -26,7 +26,7 @@ export default function RootLayout({
       <body className={cn(inter.className, "dark")}>
         <Providers>
           <Navbar />
-          <main className="h-[93vh] pt-4">{children}</main>
+          <main className="flex justify-center h-[93vh] pt-4">{children}</main>
```

Comment:
I didn’t do justify center here just in case we didn’t want things centered everywhere.

same with flex, just in case we need to do something a bit more esoteric somewhere i feel like its better to have the `main` element just control the height and padding

Reply:
I wanted to manage the width of the builder, as we are somewhat 'hacking' the width and positioning of elements to be relative to the page. Eg, the filter button is placed and adjusted (left) relatively instead of just being based on the bounding box.

If you're really against this i can move this to a wrapper for the builder page - this pattern is something i've seen more often through pages to control max-width content

Reply:
if we want to add the classes to ensure the `main` element's max width here i'm all for that. maybe even just adding `container` from tailwind would be enough.

adding `flex` and `justify-center` don't actually achieve that goal though. they are positional and layout based classes that have nothing to do with sizing at all.